### PR TITLE
Issue 27612, updated registry path

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses https://github.com/openshift/openshift-docs/issues/27612

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@kikisdeliveryservice @cgwalters @yuqi-zhang Can you please confirm this change? Thank you!

**Comment by yuqi-zhang** [comment]
This does seem to match better with the upstream docs, although the containers team probably has more insight into this. CC'ing @vrothberg and @umohnani8 for the review

**Comment by ahardin-rh** [comment]
@yuqi-zhang Thank you!

**Comment by umohnani8** [comment]
Any reason we document the MC way of making changes to these low level files? We have the cluster wide `Image` CR and the `ImageContentSourcePolicy` CR that users can use to make their registry changes and the controller updates the appropriate files on the node accordingly.

**Comment by ahardin-rh** [comment]
@umohnani8 I'm not sure exactly. Looks like this content was introduced in https://github.com/openshift/openshift-docs/pull/25823. I'm not very familiar with this area of content, but I'm happy to update the procedures with your guidance. Thank you!

File: modules/security-deploy-signature.adoc [hunk 1]
```diff
@@ -20,15 +20,15 @@ configuration is required.
...
 server. These signatures are more flexible; you can serve images from any
 container image registry and use an independent server to deliver binary
-signatures. 
+signatures.
 
 However, the `docker` transport type requires additional configuration. You must
 configure the nodes with the URI of the signature server by placing
 arbitrarily-named YAML files into a directory on the host system,
-`/etc/containers/registries.d` by default. The YAML configuration files contain a
+`/etc/containers/registries.conf.d` by default. The YAML configuration files contain a
```
Comment (vrothberg) [inline modules/security-deploy-signature.adoc:28]: In that case, the previous path was correct. Signatures go into `registries.d`.

File: modules/security-deploy-signature.adoc [hunk 2]
```diff
@@ -20,15 +20,15 @@ configuration is required.
...
 registry URI and a signature server URI, or _sigstore_:
 
-.Example registries.d file
+.Example registries.conf.d file
```
Comment (vrothberg) [inline modules/security-deploy-signature.adoc:31]: Same here.

File: modules/security-deploy-signature.adoc [hunk 3]
```diff
@@ -38,9 +38,9 @@ docker:
 
 In this example, the Red Hat Registry, `access.redhat.com`, is the signature
 server that provides signatures for the `docker` transport type. Its URI is
-defined in the `sigstore` parameter. You might name this file 
-`/etc/containers/registries.d/redhat.com.yaml` and use the Machine Config
+defined in the `sigstore` parameter. You might name this file
+`/etc/containers/registries.conf.d/redhat.com.yaml` and use the Machine Config
```
Comment (vrothberg) [inline modules/security-deploy-signature.adoc:42]: And here :)

File: modules/machineconfig-modify-registry.adoc [hunk 4]
```diff
@@ -25,7 +25,7 @@ See the `containers-registries.conf` man page for the format for the `registries
 
 .Procedure
 
-. Create a YAML file (`myregistry.yaml`) to hold the contents of the `/etc/containers/registries.d/99-worker-unqualified-search-registries.conf` file, including the encoded base64 contents for that file. For example:
+. Create a YAML file (`myregistry.yaml`) to hold the contents of the `/etc/containers/registries.conf.d/99-worker-unqualified-search-registries.conf` file, including the encoded base64 contents for that file. For example:
```
Comment (vrothberg) [inline modules/machineconfig-modify-registry.adoc:28]: And here.

**Review by vrothberg — COMMENTED** [review]
Thanks for the ping.  Some changes need to be reverted.

The naming of the two directories is unfortunate and very confusing; historical reasons. The rule of thumb is that `.conf` (TOML) files go into `registries.CONF.d`. YAMLs for signature policies etc. go into `registries.d.

File: modules/machineconfig-modify-registry.adoc (hunk 4 above)
Comment (umohnani8) [reply modules/machineconfig-modify-registry.adoc:28]: `registries.conf.d` is actually what we need here as it is talking about adding a drop-in file to  modify `unqualified-search-registries`.

Comment (umohnani8) [reply modules/machineconfig-modify-registry.adoc:28]: @ahardin-rh looks like this is a bit outdated. In 4.7, we added a new option to the cluster wide `Image` CR - `containerRuntimeSearchRegistries`. Users can use that set their configured list of unqualified-search-registries and the controller rolls out the changes to the appropriate nodes. However, we heavily advice against using unqualified-search-registries and it was documented by https://github.com/openshift/openshift-docs/pull/28152/files.

I think we should not document how someone can do this with a MC, we want users to use our CRDs for making such changes to the node. Signatures are not supported yet, so using an MC for that makes sense, but that is something we are looking to combine into the `Image` CRD in the future.

Comment (ahardin-rh) [reply modules/machineconfig-modify-registry.adoc:28]: @umohnani8 Thank you! Should I create a new PR to remove this procedure from the 4.7+ doc set then? 

FYI @mburke5678

Comment (umohnani8) [reply modules/machineconfig-modify-registry.adoc:28]: Yes please, it should be removed from the 4.7 and later docs.

Comment (ahardin-rh) [reply modules/machineconfig-modify-registry.adoc:28]: @umohnani8 Thank you! I will get this work merged and then remove the procedure from 4.7 and 4.8 in a follow-up PR. Thanks!

File: modules/machineconfig-modify-registry.adoc [hunk 5]
```diff
@@ -102,7 +102,7 @@ Starting pod/ip-10-0-141-142us-east-2computeinternal-debug ...
 [source,terminal]
 ----
 sh-4.2# chroot /host
-sh-4.4# cat /etc/containers/registries.d/99-worker-unqualified-search-registries.conf
+sh-4.4# cat /etc/containers/registries.conf.d/99-worker-unqualified-search-registries.conf
 unqualified-search-registries = ['registry.access.redhat.com', 'docker.io', 'quay.io']
 sh-4.4# exit
 ----
```
Comment (lbarbeevargas) [inline modules/machineconfig-modify-registry.adoc:108]: The update LGTM. Should these `sh-4.4#` commands be separated into individual code blocks?

File: modules/security-deploy-signature.adoc [hunk 6]
```diff
@@ -38,9 +38,9 @@ docker:
...
 In this example, the Red Hat Registry, `access.redhat.com`, is the signature
 server that provides signatures for the `docker` transport type. Its URI is
-defined in the `sigstore` parameter. You might name this file 
+defined in the `sigstore` parameter. You might name this file
 `/etc/containers/registries.d/redhat.com.yaml` and use the Machine Config
 Operator to
 automatically place the file on each node in your cluster. No service
 restart is required since policy and `registries.d` files are dynamically
-loaded by the container runtime. 
+loaded by the container runtime.
```
Comment (lbarbeevargas) [inline modules/security-deploy-signature.adoc:46]: Looks like this file is just picking up removing extra whitespace at the end of these lines. What do you think about undoing the hard line wraps in this module?

**Comment by lbarbeevargas** [comment]
A couple of minor comments, but otherwise LGTM!

File: modules/security-deploy-signature.adoc (hunk 6 above)
Comment (ahardin-rh) [reply modules/security-deploy-signature.adoc:46]: Yeah, we'll need to do some follow-up work there, but that's out of scope for this PR. Thanks!

**Pull request merged**